src/internal/observable/dom/fetch.ts

```diff
@@ -100,56 +100,70 @@ export function fromFetch<T>(
 ): Observable<Response | T> {
   const { selector, ...init } = initWithSelector;
   return new Observable<Response | T>((subscriber) => {
+    // Our controller for aborting this fetch.
+    // Any externally provided AbortSignal will have to call
+    // abort on this controller when signaled, because the
+    // signal from this controller is what is being passed to `fetch`.
     const controller = new AbortController();
-    const signal = controller.signal;
+    const { signal } = controller;
+    // This flag exists to make sure we don't `abort()` the fetch
+    // before we code is done using the response.
+    // In particular in the case where a `selector` is provided.
     let abortable = true;
```

Comment:
issue: This comment isn't quite right, IMO. The fetch is abortable for as long as the Response's lifetime is bound by this observable's lifetime. If the Response is emitted in a `next` notification, `abortable` needs to be set to `false`, as unsubscription from this observable would abort the promise from a subsequent `json()` - or whatever - call that could be made in a flattening operator.

I'd suggest a comment like this:

```ts
// This flag exists to make sure we don't `abort()` the fetch upon tearing down
// this observable after emitting a Response. Aborting in such circumstances
// would also abort subsequent methods - like `json()` - that could be called
// on the Response.
```

Reply:
That's right. Thanks, @cartant